src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -4037,6 +4072,13 @@ public void endCompensatedBlock(ForkJoinPool pool, long post) {
                     pool.endCompensatedBlock(post);
                 }
             });
-        Class<?> dep = LockSupport.class; // ensure loaded
+        defaultForkJoinWorkerThreadFactory =
+            new DefaultForkJoinWorkerThreadFactory();
+        @SuppressWarnings("removal")
+        ForkJoinPool p = common = (System.getSecurityManager() == null) ?
+            new ForkJoinPool((byte)0) :
+            AccessController.doPrivileged(new PrivilegedAction<>() {
+                    public ForkJoinPool run() {
+                        return new ForkJoinPool((byte)0); }});
```

Comment:
@AlanBateman Any recommendation as to what is ideal here with SM removed? /cc @DougLea

Reply:
I  moved this to end so that in next pass with SM removed, the static init will end with just:
common = new ForkJoinPool((byte)0).
(Which will be sure to be done after all the other static init finiahes.)

Reply:
@DougLea I must've forgotten about addressing SM separately. 👍